components/steplist/index.css

```diff
@@ -128,11 +116,23 @@ governing permissions and limitations under the License.
   &.is-selected {
     /* Focus */
     &:focus,
+    &.is-focused,
```

Comment:
We actually just removed our usage of the `.is-focused` classes so this can be removed!